web/templates/play/show.html.eex

```diff
@@ -1,2 +1,4 @@
 <%= render "_modal_keybinds.html" %>
 <div id="board-viewer" class="play-show"></div>
+
+<%= link "Explodey View", to: skin_path(@conn, :show, @game), class: "btn btn-default btn-xs view-btn" %>
```

Comment:
I think we should keep this somewhere on the page.  There's no other way to switch views on the same game, especially if there's a long list of games on the main screen.  It would be difficult to go back to the main screen and find your game.

The link's pretty unobtrusive, maybe a popup?